docs/docs/guides/tools/comparison.md

```diff
@@ -0,0 +1,113 @@
+# Comparison
+
+The Weave Comparison feature allows you to visually compare and diff code, traces, prompts, models, model configs, evaluation metrics, and evaluation predictions.  You can compare two objects side-by-side or analyze a larger set of objects to identify differences, patterns, and trends.
+
+This guide covers the steps to start a comparison and the available actions to tailor your comparison view, including baseline comparisons, numeric diff formatting, and more. 
+
+> Make sure to look at the [Usage notes](#usage-notes).
+
+## Access the Comparison view
+
+1. In the sidebar, select the type of object you'd like to compare (e.g. **Traces**, **Models**, etc.).
+2. Select the checkbox for each object that you want to compare.
+3. Click **Compare**. The Comparison view displays. Now, you can refine your view using the [available actions](#available-actions).
+
+## Available actions
+
+In the Comparison view, you have multiple actions available, depending on how many objects are being compared.
+
+### Change the diff display
+
+By default, **Diff only** is set to off. To filter the table rows so that only changed rows are displayed, toggle **Diff only** on. 
+
+### Display side-by-side 
+
+> This option is only available when comparing two objects, or a [subset of a multi-object comparison](#compare-a-subset-of-objects).
```

Comment:
I might say "pair from a multi-object comparison" just because you can't select subsets that aren't cardinality 2 but I don't feel strongly about it if you think this is clearer.

Reply:
After rereading, your phrasing makes more sense. Updated all mentions and header in https://github.com/wandb/weave/pull/3124/commits/65d43eb765868268559f8573347bdcc4a3302dcd